tailwind.config.js

```diff
@@ -173,6 +173,22 @@ module.exports = {
       opacity: {
         '10': '0.1',
         '60': '0.6'
+      },
+      keyframes: {
+        appear: {
+          '0%': {
+            'opacity': '0',
+            'pointer-events': 'none'
+          },
+          '100%': {
+            'opacity': '1',
+            'pointer-events': 'auto'
+          }
+        }
+      },
+      animation: {
+        fadeIn:
+          'appear 50ms linear 400ms both'
```

Comment:
Usually tailwind rules are kebab-cased. I'd say it's nice to follow their convention. Also, we could put the string on the same line as the key
```suggestion
        'fade-in': 'appear 50ms linear 400ms both'
```